src/content/en/2019/javascript.md

```diff
@@ -7,215 +7,182 @@ reviewers: [rviscomi, obto, paulcalvano, mathiasbynens]
-usable in many environments, not only web browsers.
+The language specification itself, along with many community-built libraries and frameworks used by developers around the world, has changed and evolved ever since the language was created in 1995. JavaScript implementations and interpreters have also continued to progress, making the language usable in many environments, not only web browsers.
 
-The [HTTP Archive](https://httparchive.org/) crawls [millions of
-URLs](https://httparchive.org/reports/state-of-the-web#numUrls) every month and runs them through a
-private instance of [WebPageTest](https://webpagetest.org/) to store key information of every page
-(see [methodology](./methodology)). In the context of JavaScript, HTTP Archive provides extensive
-information on the usage of the language for the entire web. This chapter consolidates and analyzes
-many of these trends. 
+The [HTTP Archive](https://httparchive.org/) crawls [millions of URLs](https://httparchive.org/reports/state-of-the-web#numUrls) every month and runs them through a private instance of [WebPageTest](https://webpagetest.org/) to store key information of every page (see [methodology](./methodology)). In the context of JavaScript, HTTP Archive provides extensive information on the usage of the language for the entire web. This chapter consolidates and analyzes many of these trends.
 
-# How much JavaScript do we use?
+## How much JavaScript do we use?
 
-JavaScript is the most costly resource we send to browsers, having to be downloaded, parsed,
-compiled and then finally executed. Although browsers have significantly decreased the time it takes
-to parse and compile scripts,
-[download and execution have become the most expensive stages](https://v8.dev/blog/cost-of-javascript-2019)
-when JavaScript is processed by a web page.
+JavaScript is the most costly resource we send to browsers, having to be downloaded, parsed, compiled and then finally executed. Although browsers have significantly decreased the time it takes to parse and compile scripts, [download and execution have become the most expensive stages](https://v8.dev/blog/cost-of-javascript-2019) when JavaScript is processed by a web page.
 
-Sending smaller JavaScript bundles to the browser is the best way to reduce download times, and in
-turn improve page performance. **But how much JavaScript do we really use?**
+Sending smaller JavaScript bundles to the browser is the best way to reduce download times, and in turn improve page performance. **But how much JavaScript do we really use?**
 
 Here's a chart showing the amount of bytes shipped for different percentiles:
 
 ```<insert graphic of metric 01_01a>```
+<figure>
+	<iframe aria-labelledby="fig1-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1974602890&format=interactive"></iframe>
```

Comment:
For each iframe we should also have an img equivalent for mobile. See #237 ("interactive figures")